bundles/org.openhab.binding.dwdpollenflug/src/main/resources/ESH-INF/thing/region.xml

```diff
@@ -0,0 +1,162 @@
+			</channel-group>
+			<channel-group id="updates" typeId="updates"/>
+		</channel-groups>
+		
+		<properties>
+			<property name="region_id"/>
+			<property name="region_name"/>
+			<property name="partregion_name"/>
+		</properties>
+		
+		<config-description>
+			<parameter name="regionID" type="integer" required="true">
+				<label>Region</label>
+				<description>The partregion or region that should be reported.</description>
+				<options>
+					<option value="11">Inseln und Marschen</option>
+					<option value="12">Geest, Schleswig-Holstein und Hamburg</option>
+					<option value="20">Mecklenburg-Vorpommern</option>
+					<option value="31">Westl. Niedersachsen/Bremen</option>
+					<option value="32">Östl. Niedersachsen</option>
+					<option value="41">Rhein.-Westfäl. Tiefland</option>
+					<option value="42">Ostwestfalen</option>
+					<option value="43">Mittelgebirge NRW</option>
+					<option value="50">Brandenburg und Berlin</option>
+					<option value="61">Tiefland Sachsen-Anhalt</option>
+					<option value="62">Harz</option>
+					<option value="71">Tiefland Thüringen</option>
+					<option value="72">Mittelgebirge Thüringen</option>
+					<option value="81">Tiefland Sachsen</option>
+					<option value="82">Mittelgebirge Sachsen</option>
+					<option value="91">Nordhessen und hess. Mittelgebirge</option>
+					<option value="92">Rhein-Main</option>
+					<option value="101">Rhein, Pfalz, Nahe und Mosel</option>
+					<option value="102">Mittelgebirgsbereich Rheinland-Pfalz</option>
+					<option value="103">Saarland</option>
+					<option value="111">Oberrhein und unteres Neckartal</option>
+					<option value="112">Hohenlohe/mittlerer Neckar/Oberschwaben</option>
+					<option value="113">Mittelgebirge Baden-Württemberg</option>
+					<option value="121">Allgäu/Oberbayern/Bay. Wald</option>
+					<option value="122">Donauniederungen</option>
+					<option value="123">Bayern n. der Donau, o. Bayr. Wald, o. Mainfranken</option>
+					<option value="124">Mainfranken</option>
+				</options>
+			</parameter>
+		</config-description>
+	</thing-type>
+	
+	<channel-group-type id="pollentype">
+		<label>Pollen type group</label>
+		<description>For each pollen type there are three channels for today, tomorrow and day after tomorrow</description>
+		<channels>
+			<channel id="today" typeId="today" />
+			<channel id="tomorrow" typeId="tomorrow" />
+			<channel id="dayafter_to" typeId="dayafter_to" />
+		</channels>
+	</channel-group-type>
+	
+	<channel-type id="today">
+		<item-type>String</item-type>
+		<label>Today</label>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="tomorrow">
+		<item-type>String</item-type>
+		<label>Tomorrow</label>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="dayafter_to">
+		<item-type>String</item-type>
+		<label>Day after tomorrow</label>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-group-type id="updates">

```

Comment:
If these values are available on the bridge, there is no need to add them to each thing

Reply:
Moved to bridge instead of properties (see above)